src/sbvr-api/sbvr-utils.coffee

```diff
@@ -723,6 +734,39 @@ exports.getAbstractSqlModel = getAbstractSqlModel = (request) ->
 	request.abstractSqlModel ?= abstractSqlModels[request.vocabulary]
 	return request.abstractSqlModel
 
+exports.getAffectedIds = ({ req, request, tx }) ->
+	# We reparse to make sure we get a clean odataQuery, without permissions already added
+	# And we use the request's url rather than the req for things like batch where the req url is ../$batch
+	uriParser.parseOData({
+		method: request.method
+		url: "/#{request.vocabulary}#{request.url}"
+	})
+	.then (request) ->
+		abstractSqlModel = getAbstractSqlModel(request)
+		resourceName = resolveSynonym(request)
+		resourceTable = abstractSqlModel.tables[resourceName]
+		if !resourceTable?
+			throw new Error('Unknown resource: ' + request.resourceName)
+		idField = resourceTable.idField
+
+		_.set(request.odataQuery, [ 'options', '$select' ], { 'properties': [ { 'name': idField } ] })
```

Comment:
How about dropping the single quotes from `'properties'` & `'name'` ?
Multiline formatting could also help IMO